src/mechaclient.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, *args, **kwargs):
         """
         LOG.debug(f"Connecting to channels...")
         # join a channel
-        for channel in config["irc"]["channels"]:
+        for channel in self.config["irc"]["channels"]:
```

Comment:
[error] this should now point to `self._config` as the public `.config` property was removed.